### PR TITLE
launch modal with player

### PR DESCRIPTION
**Pull request by tyard**

### CODE REVIEW

File: css/bcgs_ford_main.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+#modal-container {
+    background-color: #fff;
+    margin: 80px auto 0px auto;
+    width: 800px;
+    height: 500px;
+}
+
+#modal-player .video-js {
+    width: 100%;
```
Comment (bceacuna) [inline css/bcgs_ford_main.css]: shouldn't we prefix all our stuff with bcc-?

File: lib/bcgs_ford_main.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    initial drop
+
+*/
+(function(window, document) {
+
+    "use strict";
+    
+    // replace name of plugin, then proceed to constructor function
+    var NAME = "FordPlayer",
+
```
Comment (bceacuna) [inline lib/bcgs_ford_main.js]: are we going to use our new naming convention on this project? bccFordPlayer.

File: lib/bcc_core.js [hunk 3]
```diff
@@ -0,0 +1,378 @@
+/**
+
+Copyright(c) 2014 Brightcove, Inc.
+
```
Comment (bceacuna) [inline lib/bcc_core.js]: We need to update our template. Copyright(c) 2015 Brightcove, Inc.

File: lib/bcc_core.js [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+                    find = function(selector) {
+                        return document.querySelector(selector);
+                    },
+
+                    /**
+                     * Creates an event dispatching model of set properties
+                     * with explicit getters and setters.
+                     */
+                    Model = function(p) {
+                        var properties = {};
```
Comment (bceacuna) [inline lib/bcc_core.js]: Can you provide an example of how to use this method?

File: css/bcgs_ford_main.css (hunk 1 above)
Comment (tyard) [inline css/bcgs_ford_main.css]: It is only necessary for our reusable components when we don't know the projects they will be used in. For a customer-specific project it isn't necessary, especially in a case like this.

File: lib/bcgs_ford_main.js (hunk 2 above)
Comment (tyard) [inline lib/bcgs_ford_main.js]: I think only necessary for reusable components. bcc shouldn't be a prefix for client-specific code.

File: lib/bcc_core.js (hunk 4 above)
Comment (tyard) [inline lib/bcc_core.js]: Sure, I will update the comment. But in the meantime:
```
var model = new bcc.Model({
    state:0, view:null
});
model.bind(bcc.events.CHANGE, function(event) {
  console.log( model.getState() );
}
model.setState(1);

**Pull request merged**